assets/src/edit-story/components/panels/link/index.js

```diff
@@ -99,15 +101,33 @@ function LinkPanel({ selectedElements, pushUpdateForObject }) {
     if (!isValidUrl(urlWithProtocol)) {
       return;
     }
+    const invalidLinkMessage = __('This is an invalid link!', 'web-stories');
```

Comment:
Just _"This"_? Maybe we should show the url in the snackbar too?

```js
    const invalidLinkMessage = __('The following is not a valid link: %s', 'web-stories');
```

And then:

```js
          showSnackbar({
            message: sprintf(invalidLinkMessage, url),
          });
```

Reply:
Sure! I will update.

Reply:
Is it the expected result @barklund ? What about that blank space? 

![Kapture 2020-03-31 at 18 16 43](https://user-images.githubusercontent.com/1680157/78075975-e34f0600-737b-11ea-93da-abab33c33a4f.gif)

Reply:
Looks good to me, but perhaps validate with @pbakaus or @samitron7.

Reply:
I don't really like the URL there tbh, feels uncessary and makes the tooltip bigger than it should be